tests/StrictObjects/SuggesterTest.phpt

```diff
@@ -55,14 +57,20 @@ class SuggesterTest extends \Tester\TestCase
 	}
 
 	/**
+	 * @param string|null $expected
```

Comment:
In that case you could duplicate the rule, change it to your liking and use it instead.